### PR TITLE
Add Ontario Co-op Requirements!

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: 2018-december-retreat/2018-11-28-retreat-playbook.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ To ensure the sync time we have together is productive, we are asking all meetin
 
 - Sign up to [Loomio](https://loomio.cryptography.dog) and fill out your profile :v: 
 - Read essential and selected [Readings](#Readings)
-- Complete the [Solo Visioning Exercise](#Solo-Visioning-Exercise)
+- Complete the [Solo Visioning Exercise](#Solo-Visioning-Ehttps://github.com/hyphacoop/organizing/blob/add-retreat-docsxercise)
```
Comment (patcon) [inline 2018-december-retreat/2018-11-28-retreat-playbook.md]:
```suggestion
- Complete the [Solo Visioning Exercise](#Solo-Visioning-Exercise)
```

I think this is a typo?

File: 2018-december-retreat/ontario-coop-requirements.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
+# Ontario Co-op Requirements
```
Comment (patcon) [inline 2018-december-retreat/ontario-coop-requirements.md]: What's the source of this doc? Can we link it? (Sorry, tried searching some key phrases, but can't find it online)

File: 2018-december-retreat/ontario-coop-requirements.md [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+---
+
+# Reference Material
+
+## Worker Co-ops
+
+- A worker co‐op exists primarily to create employment for its members.
+- All members of a worker co‐operative must be employees, but not all employees must be members... At least 75% of all employees must be members and at least 75% of permanent, full-time employees must be members.
+- > [name=patcon] was curious what this meant in practice, so [sketched out](Diagramming co-op membership rules https://imgur.com/gallery/AmdyFNr)
```
Comment (patcon) [inline 2018-december-retreat/ontario-coop-requirements.md]:
```suggestion
- > [**@patcon**](https://github.com/patcon) was curious what this meant in practice, so [sketched out](https://imgur.com/gallery/AmdyFNr) a diagramming of co-op membership rules
```

File: 2018-december-retreat/ontario-coop-requirements.md (hunk 2 above)
Comment (dcwalk) [reply 2018-december-retreat/ontario-coop-requirements.md]: It is linked from the retreat doc, this was the research doc we compiled which lives as a hackmd: https://hackmd.io/GNMLlqWpQe2LREf6umqPhw

Comment (dcwalk) [reply 2018-december-retreat/ontario-coop-requirements.md]: Do we want to link from a hackpad?

Comment (dcwalk) [reply 2018-december-retreat/ontario-coop-requirements.md]: Oh, @patcon -- it's the one I linked from above. I think I may misunderstand what you meant by source, though much comes from the Appendicies and Reference material linked throughout

**Comment by dcwalk** [comment]
I'm going to let this sit for a few days them merge in as it is simply moving a pad we researched from hackmd into github.

File: 2018-december-retreat/ontario-coop-requirements.md (hunk 2 above)
Comment (patcon) [reply 2018-december-retreat/ontario-coop-requirements.md]: heh yeah, thought it was from elsewhere and just reformatted to markdown. Sounds like it's an assemblage though :) 🙌

Comment (patcon) [reply 2018-december-retreat/ontario-coop-requirements.md]:
```suggestion
---
source: https://hackmd.io/GNMLlqWpQe2LREf6umqPhw
---
# Ontario Co-op Requirements
```

Thoughts on this style? I know we shouldn't expect them to change in hackmd anymore, but feels nice to keep track of where it was?

Comment (dcwalk) [reply 2018-december-retreat/ontario-coop-requirements.md]: I'm fine with following YAML conventions, though given my difficulty interpreting the word "source" I'd say it is a little ambiguous...?

tbh I don't think linking to hackmds as a "source" adds much value over time--anticipating they eventually dissolve. I think metadata that provides descriptive context will age better and not wind up as clutter (e.g., date created, authors, event short description or purpose).

Comment (dcwalk) [reply 2018-december-retreat/ontario-coop-requirements.md]: Why do you see a specific link as useful?

Comment (benhylau) [reply 2018-december-retreat/ontario-coop-requirements.md]: I definitely don't think there is value in persisting hackmd sources that are meant to be disposable in the first place. If anything, it makes the reader more confused. I'd even go to the hackmd and delete the text, and replace it with a link to the github copy.

Comment (patcon) [reply 2018-december-retreat/ontario-coop-requirements.md]: I think deleting them, and linking github is best for now, if we don't want to use the hackmd docs after creating our PRs.

Reason I was leaning toward tracking it: At one point, I thought that a nightly task might be nice, that could gather changes in hackmds after the fact, and poof open new PRs for any additional changes. Tracking the source url seemed helpful if we wanted laypersons to be able to participate in notes clean-up and review. The idea was that hackmd was friendlier for external participants than our GitHub process. This was also related to https://github.com/hyphacoop/worker-coop-scripts/issues/1.

But that's nonexistent, so happy to revisit later if/when there's a need :)

Comment (dcwalk) [reply 2018-december-retreat/ontario-coop-requirements.md]: Okay, then I'll merge as-is

**Pull request merged**